website/docs/r/cluster_outage_simulation.html.markdown

```diff
@@ -0,0 +1,69 @@
+---
+layout: "mongodbatlas"
+page_title: "MongoDB Atlas: cluster_outage_simulation"
+sidebar_current: "docs-mongodbatlas-resource-federated-database-instance"
+description: |-
+    Provides a Cluster Outage Simulation resource.
+---
+
+# Resource: mongodbatlas_cluster_outage_simulation
+
+`mongodbatlas_cluster_outage_simulation` provides a Cluster Outage Simulation resource.
```

Comment:
```suggestion
`mongodbatlas_cluster_outage_simulation` provides a Cluster Outage Simulation resource. For more details see https://www.mongodb.com/docs/atlas/tutorial/test-resilience/simulate-regional-outage/

Test Outage on Minority of Electable Nodes - Select fewer than half of your electable nodes. This operation is supported via Terraform Provider. 

Test Outage on Majority of Electable Nodes - Select at least one more than half of your electable nodes and keep at least one electable node remaining. This operation not supported via Terraform Provider. Instead use Atlas UI instead. 
```
Pending confirm on above change, but please cleanly define Minority vs Majority in docs (feel free to edit above)